### DISCUSSION TITLE
Appveyor rate limiting

### DISCUSSION THREAD

**Discussion by vbjay** [gitextensions/gitextensions#11147] [General]
https://ci.appveyor.com/project/gitextensions/gitextensions/builds/47779477#L278  Probably need to set up build to use a github token just for our project.

**Comment by gerhardol** [comment]
For now, the builds have to be restarted manually

I do not have privileges to configure the AppVeyor app to add a token, which I believe is required.
When that is done, the token can be added to AppVeyor environment (as `GITHUB_TOKEN`?), be in the AppVeyor environment, then it should be used in the PM download script.
